src/udgl/menu/MenuButton.svelte

```diff
@@ -35,18 +35,26 @@ let button;
   padding: var(--space-1h);
   padding-left: var(--space-base);
   padding-right: var(--space-base);
-  font-size: var(--text-02);
+  border-radius: var(--space-1h);
+  font-size: calc(var(--space-base) * 1.5);
   border: none;
-  background-color: hsla(240,50%,80%,.3);
-  color: white;
+  border: 1px solid var(--cool-gray-200);
+  color: var(--digital-blue-700);
+  background-color: var(--cool-gray-subtle);
+  box-shadow: 0px 2px 2px rgba(0,0,0,.05);
+  transition: 100ms;
+  cursor: pointer;
+}
+
+.activating-button:hover {
```

Comment:
Will these interfere with the GP Button work by @openjck ?

Reply:
This is a stopgap since his PR has not been revised yet. I didn't want to block on it.

Reply:
This all said, I'll move this `MenuButton` component out of `udgl` and into `components` for now.

Reply:
Don't worry about me; I don't mind rebasing.